conda-store-server/conda_store_server/action/generate_lockfile.py

```diff
@@ -13,19 +15,39 @@ def action_solve_lockfile(
     conda_command: str,
     specification: schema.CondaSpecification,
     platforms: typing.List[str] = [conda_utils.conda_platform()],
```

Comment:
I propose we do this later in one go, to avoid unrelated issues with linters etc. I've opened an issue: https://github.com/conda-incubator/conda-store/issues/711